src/gatsby-plugin-theme-ui/index.js

```diff
@@ -413,6 +417,23 @@ export default {
               backgroundColor: '#13ADC7'
             }
           }
+        },
+        OtherPopup: {
```

Comment:
Examples of how it looks: 

![image](https://user-images.githubusercontent.com/43496356/153961294-093108b2-2a9b-4e0f-9261-abb8c9d88247.png)

![image](https://user-images.githubusercontent.com/43496356/153961333-bdc266b4-d277-4235-a040-ff8bfbc3ad5d.png)

![image](https://user-images.githubusercontent.com/43496356/153961383-9a26012f-5de8-404a-abb0-4d2895dedcb9.png)